### ISSUE TITLE
History page is the default page when reopen the app

### ISSUE THREAD

**Issue by Zeyadas** [koreader/koreader#7720]
* KOReader version: latest 5.2021
* Device: Samsung galaxy Tab A (T-285) android tablet 

When switching the app to take notes with a separate notes app (using the resent soft key) and reopen koreader it opens on history page.
I noticed that from the previous version (4.2021).

**Comment by pazos** [comment]
The app can get killed by Android if there's no enough memory for the foreground app. We usually never reach this on android devices, but my test devices are a heavy limited Android Emulator (VM Heap 16MB, total memory: 256MB) without any other service running and no google play software installed and a modern Android tablet with loads of RAM.

Real life usage on legacy devices with tons of background services might be different. Maybe a device restart make your device less ram-hungry.

Probably nothing to do with us. Anyways please report a log using tomorrow nightly.

**Comment by Frenzie** [comment]
The device seems to have a mere 1.5 GB, but I'd still expect that to suffice, generally speaking.

**Comment by Zeyadas** [comment]
> The app can get killed by Android if there's no enough memory for the foreground app.

Yes it is, I think that's the case. This option was activated by me to make the battery last longer and didn't notice the issue until I had to use a notes app. 
It totally solved after I deactivated it. 
So many thanks guys. 

![IMG_20210522_002252_397](https://user-images.githubusercontent.com/70109540/119199814-69547180-ba94-11eb-8372-899668e6708a.jpg)
